README.md

```diff
@@ -14,27 +14,74 @@ You can deploy these samples directly through the Azure Portal or by using the s
 
 To deploy a sample using the Azure Portal, click the **Deploy to Azure** button found in the README.md of each sample.
 
-To deploy the sample via the command line (using [Azure PowerShell or the Azure CLI](https://azure.microsoft.com/en-us/downloads/)) you can use the scripts.
+To deploy the sample via the command line (using [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/overview) or the [Azure CLI 1.0](https://docs.microsoft.com/en-us/azure/cli-install-nodejs)) you can use the scripts below.
 
-Simply execute the script and pass in the folder name of the sample you want to deploy.  For example:
+Simply execute the script and pass in the folder name of the sample you want to deploy.  
 
+For example:
+
+### PowerShell
 ```PowerShell
 .\Deploy-AzureResourceGroup.ps1 -ResourceGroupLocation 'eastus' -ArtifactStagingDirectory '[foldername]'
 ```
+
+### Bash
 ```bash
 azure-group-deploy.sh -a [foldername] -l eastus
 ```
+
+### Bash on Ubuntu on Windows
+
+Run the following the first time to get all prerequisites installed:
```

Comment:
I'm looking for lowest dev friction here as possible.

1. That link does not show how to install npm or node, which are not installed in Linux subsystem by default. There's a link on that page, but now we are 3 links deep into node install and their install page scrolls forever.  It doesn't hurt to include the exact steps here. Without the instructions inline you will increase the developer start time by at least 5 minutes probably more.

2. I had never heard of dos2unix before either, but without running it I get the following error (on Linux Subsystem). 

If you see the following error: "syntax error near unexpected token `$'in\r''", make sure you run the dos2unix command above.